src/components/organisms/SectionContents/facebook.jsx

```diff
@@ -3,6 +3,12 @@ import * as React from 'react'
 //** Additional Frameworks */
 import { FacebookProvider, EmbeddedPost } from 'react-facebook';
 
+//** Helpers */
+//** Personalization */
+import Text from "../../helper/Text";
+import { renderToString } from 'react-dom/server';
+import ReactHtmlParser from 'react-html-parser'; 
+
 /**
  * Section: Facebook
  */
```

Comment:
This is disgusting drug-induced crap, and should die
To put it with Linus words.